library/constant_time_internal.h

```diff
@@ -236,63 +236,6 @@ void mbedtls_ct_memcpy_offset(unsigned char *dest,
                               size_t offset_max,
                               size_t len);
 
-/** Compute the HMAC of variable-length data with constant flow.
```

Comment:
It's still included in `constant_time.c` so that we can save a bit of code-size (by not building `mbedtls_ct_memcpy_offset`). So really, it should probably be here too but I ended up removing it from the header for some reason.

Reply:
I'm not sure this is a good idea. If there are functions that are not called at all, the linker will garbage-collect them. Guards are not really useful here, but on the other hand they have a downside: including an SSL file from a file that belongs to libmbedcrypto is a layering violation, and exactly the kind of thing that's going to get us in trouble when we split repos.

Reply:
IMO all of the const-time primitives should all live together in `constant_time.c` and `constant_time.impl` for maintainability. So I'd be more inclined to simply lose the guards and the `#include` - I agree that these should go.